.buildkite/pipeline.yml

```diff
@@ -149,6 +148,7 @@ steps:
       system: ${macos}
 
   - block: "macOS package"
+    depends_on: macos-nix
     if: 'build.branch != "master"'
```

Comment:
I just noticed that our decision on Continuous Integration states that we want to build macOS artifacts `pre-merge`.

<img width="645" alt="release-artifacts" src="https://user-images.githubusercontent.com/564592/228763408-d5280793-3e8d-43a4-9d52-b2a4a56ee20a.png">

Hence we want to change this line:

```suggestion
    if: '(build.branch != "staging") && (build.branch != "trying") && (build.branch != "master")'
```